main.go

```diff
@@ -1,48 +1,39 @@
 package gocolor
 
-var defalt string = "\u001b[0m"
+var _default string = "\x1b[0m"
 
-func Defalt(text string) string {
-	result := "\u001b[0m" + text
-	return result
+func Default(text string) string {
+	return "\x1b[0m" + text
```

Comment:
```suggestion
// Deprecated: Use Default
func Defalt(text string) string {
	return Default(text)
}

func Default(text string) string {
	return "\x1b[0m" + text
```

どう思いますか？
これでbreaking changeの変更じゃないでしょう。

（「breaking change」を日本語で書き方はわらなくてすみません。）